smokefree/sites/all/modules/custom/icfi_section_landing/icfi_section_landing.box.inc

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @file
+ * icfi_section_landing.box.inc
+ */
+
+/**
+ * Implements hook_default_box().
+ */
+function icfi_section_landing_default_box() {
+  $export = array();
+
+  $box = new stdClass();
+  $box->disabled = FALSE; /* Edit this to true to make a default box disabled initially */
+  $box->api_version = 1;
+  $box->delta = 'dip_hurts';
+  $box->plugin_key = 'simple';
+  $box->title = '<none>';
+  $box->description = 'dip-hurts';
+  $box->options = array(
+    'body' => array(
+      'value' => '<iframe src="/sites/interactives/dip-hurts/" style="border:none;height: 100vh;" class="col-sm-12"></iframe>',
```

Comment:
I understand this code was written by an external contractor and we have been advised not to re-write the entire code, however, I feel we should try to improve some aspects of the code. 

Do we need to use iframes to implement this? Can we have the contents of the HTML page (/sites/interactives/dip-hurts/index.html) in this block and avoid loading it as an iframe altogether?

Using an iframe here would load an additional (hardcoded) page which will mess with our Varnish caching. Additionally, it will also mess with SEO. 

Edit: Same applies for the following
- https://github.com/CBIIT/smokefree/pull/7/files#diff-21e580cf030021c270d7b0e565b9dde7R38
- https://github.com/CBIIT/smokefree/pull/7/files#diff-21e580cf030021c270d7b0e565b9dde7R54

Reply:
The major reason I had to use iframe is because I tried put one interactive as a block and site css apply to the block and messed up the interactive.

Reply:
It should definitely be possible to move this functionality into its block.

From what Amy described, it appears like this will only be displayed on the the three pages and now, if possible we should refrain from adding more development time to this task. So, there is no need to change the implementation.